apertium_apy/handlers/per_word.py

```diff
@@ -5,12 +5,13 @@
 from tornado import gen
 
 from apertium_apy.handlers.base import BaseHandler
-from apertium_apy.utils import apertium, run_async_thread, remove_dot_from_deformat, to_alpha3_code
```

Comment:
I think the util `run_async_thread` be deleted?

Reply:
Oh, I removed that. I didn't understand why it used both multiprocessing *and* threads. Actually, wouldn't it be simpler to just do the whole thing like 

```
async def process_per_word(…): 
    intermediateresult = await apertium(…)

result = await process_per_word(…)
handle_output(result)
```
?

The multiprocessing thing doesn't seem very useful when 1) the python code itself is not the bottleneck and 2) we're running things sequentially anyways.

Reply:
asyncio.run isn't in 3.6, so switched to async/await – seems to work :)

Reply:
Apologies, I missed a word, I meant:

> I think the util run_async_thread *can* be deleted?

since it's dead.

I agree that there's some misguided stuff going on - thanks for cleaning up.